SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -2221,6 +2223,48 @@ void CIncEulerSolver::Source_Residual(CGeometry *geometry, CSolver **solver_cont
     }
   }
 
+  if (radiation) {
```

Comment:
One more question @rsanfer, this is not compatible with the compressible solvers?

Reply:
So far it is not, as I have only tested it for the incompressible. In theory it should not be too hard to adapt it, though, so we can merge this in now and I can look at the specifics and add a patch for the following release.

Reply:
Should we throw an error in case someone tries to use radiation+compressible flow? (Maybe it is there already and I just did not notice)